### PR TITLE
raidboss: convert most hw triggers to output strings

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/raidboss/data/03-hw/raid/a4s.js [hunk 1]
```diff
@@ -48,15 +50,18 @@
...
-        return {
-          en: 'Stun ' + matches.target,
-          de: 'Unterbreche ' + matches.target,
-          fr: 'Stun ' + matches.target,
-          ja: 'スタン: ' + matches.target,
-          cn: '眩晕' + matches.target,
-          ko: matches.target + '기절 시키기',
-        };
+      alertText: function(data, matches, output) {
+        return output.text({ param: matches.target });
```
Comment (panicstevenson) [inline ui/raidboss/data/03-hw/raid/a4s.js:54]: needless nit: `param` could be a bit more descriptive

File: ui/raidboss/data/03-hw/raid/a9s.js [hunk 2]
```diff
@@ -131,30 +131,34 @@
...
-        };
+        return output.getBehind();
       },
-      infoText: function(data) {
+      infoText: function(data, _, output) {
         if (data.mainTank != data.me)
           return;
-        return {
+        return output.scraplineOnYou();
+        // ...probably, we hope...
```
Comment (panicstevenson) [inline ui/raidboss/data/03-hw/raid/a9s.js:143]: This is odd

**Review by panicstevenson — APPROVED** [review]

File: ui/raidboss/data/03-hw/raid/a4s.js (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a4s.js:54]: Thanks! Changed to `${name}`.

File: ui/raidboss/data/03-hw/raid/a9s.js (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a9s.js]: Removed.  It was intended to be a "you might not be the main tank anymore, but hopefully you still are! and if you're not, probably everybody else is in trouble!!"

**Pull request merged**